code/controllers/configuration.dm

```diff
@@ -102,6 +102,8 @@
 	var/use_age_restriction_for_jobs = 0 //Do jobs use account age restrictions? --requires database
 	var/use_account_age_for_jobs = 0	//Uses the time they made the account for the job restriction stuff. New player joining alerts should be unaffected.
 	var/see_own_notes = 0 //Can players see their own admin notes (read-only)? Config option in config.txt
+	var/note_fresh_days

```

Comment:
Comments here ploxkthxbye

Reply:
probably needs a default value aswell

Reply:
nope, the default is set to null intentionally. 0 = start fading out immediately, null = disabled.